connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -0,0 +1,136 @@
+import json
+import logging
+import re
+import subprocess  # nosec
+from connaisseur.validators.interface import ValidatorInterface
+from connaisseur.image import Image
+from connaisseur.exceptions import (
+    CosignError,
+    CosignTimeout,
+    NotFoundException,
+    ValidationError,
+    UnexpectedCosignData,
+    InvalidFormatException,
+)
+from connaisseur.crypto import load_key
+
+
+class CosignValidator(ValidatorInterface):
+    name: str
+    keys: list
+
+    def __init__(self, name: str, keys: list, **kwargs):
+        self.name = name
+        self.keys = keys
+
+    def __get_key(self, key_name: str = None):
+        try:
+            key_name = key_name or "default"
+            key = next(key["key"] for key in self.keys if key["name"] == key_name)
+            return "".join(key)
```

Comment:
The `key_name` assignment can go before the `try` block, and the `return` statement can go after the `except` block